samples/openthread/cli/README.rst

```diff
@@ -177,6 +186,78 @@ To test diagnostic commands, complete the following steps:
       last received packet: rssi=-30, lqi=255
       Done
 
+Testing Thread 1.2 features
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To test some Thread 1.2 features, complete the following steps:
+
+#. Make sure both development kits are programmed with the CLI sample.
```

Comment:
Is it necessary for both dev kits to have 1.2 enabled or only one is needed? Maybe we should mention it here.

Reply:
Both of them.  What about this?

```suggestion
#. Make sure both development kits are programmed with the CLI sample with the Thread 1.2 variant.
```